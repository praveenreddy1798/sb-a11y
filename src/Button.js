import { Button as Btn } from '@chakra-ui/react';

export const Button = ({ colorScheme }) => {
  return <Btn colorScheme={colorScheme}>Button</Btn>;
};
