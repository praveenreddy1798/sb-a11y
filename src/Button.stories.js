import React from 'react';
import { Button } from './Button';

export default {
  component: Button,
  title: 'Components/Button',
};

const Template = (args) => <Button {...args} />;

export const Primary = Template.bind({});

Primary.args = {
  colorScheme: 'brand',
};

export const Secondary = Template.bind({});

Secondary.args = {
  colorScheme: 'red',
  // colorScheme: 'red.400',
};
