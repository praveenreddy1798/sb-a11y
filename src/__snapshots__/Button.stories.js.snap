// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components/Button Primary smoke-test 1`] = `
Object {
  "checked": undefined,
  "children": Array [
    Object {
      "checked": undefined,
      "children": undefined,
      "name": "Button",
      "pressed": undefined,
      "role": "button",
      "value": undefined,
    },
  ],
  "name": "components-button--primary",
  "pressed": undefined,
  "role": "WebArea",
  "value": undefined,
}
`;

exports[`Components/Button Secondary smoke-test 1`] = `
Object {
  "checked": undefined,
  "children": Array [
    Object {
      "checked": undefined,
      "children": undefined,
      "name": "Button",
      "pressed": undefined,
      "role": "button",
      "value": undefined,
    },
  ],
  "name": "components-button--secondary",
  "pressed": undefined,
  "role": "WebArea",
  "value": undefined,
}
`;
